IGI.Lab2/src/CarPark.ConsoleUI/ConsoleService/MenuService/MenuConsoleService.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using CarPark.ConsoleUI.Interfaces;
+
+namespace CarPark.ConsoleUI.ConsoleService.MenuService
```

Comment:
You can reduce namespace to that:
_CarPark.ConsoleUI.ConsoleService**s**_

PS: Fix namespace and folder structure for all console services.

Reply:
fixed